src/main/java/levy/art/ImageFrame.java

```diff
@@ -0,0 +1,28 @@
+package levy.art;
+
+import javax.swing.*;
+import java.awt.*;
+import java.net.URL;
+
+public class ImageFrame extends JFrame {
+    public ImageFrame(String title, String artist, String imageUrl) {
+        setTitle(title + " by " + artist);
+        setDefaultCloseOperation(DISPOSE_ON_CLOSE);
+        setSize(800, 600);
+
+        try {
+            URL url = new URL(imageUrl);
+            ImageIcon icon = new ImageIcon(
+                    new ImageIcon(url).getImage().getScaledInstance(800, -1, Image.SCALE_SMOOTH)
+            );
```

Comment:
This is not how I showed you to download and resize images.

Reply:
I know I had to change it around because checkstyles got upset that the line was too long

Reply:
> I know I had to change it around because checkstyles got upset that the line was too long 

Then break up the line...

Reply:
when i did it wanted me to move it back